src/libraries/Microsoft.Extensions.Configuration/src/ConfigurationManager.cs

```diff
@@ -106,9 +92,9 @@ IConfigurationBuilder IConfigurationBuilder.Add(IConfigurationSource source)
 
         void IConfigurationRoot.Reload()
         {
-            lock (_providerLock)
+            using (var reference = _providerManager.GetReference())
```

Comment:
Bad `var` usage. Repeated elsewhere in this code.

Reply:
My find/replace failed here because it's `using (var ...`  instead of `using var ...`. Did I miss this somewhere else?

Reply:
I added a new review with each one marked.